aries-site/src/examples/templates/persistent-navigation/HeaderNavExample.js

```diff
@@ -0,0 +1,256 @@
+
+  return (
+    <ResponsiveContext.Provider value={mobile && 'small'}>
+      <ResponsiveContext.Consumer>
+        {size => (
+          <AppContainer activeItem={activeItem}>
+            <Header fill="horizontal" pad="medium">
+              <AppIdentity name={pages[activeItem].name} />
+              <Box direction="row" gap="medium">
+                <MainNavigation
+                  activeItem={activeItem}
+                  setActiveItem={setActiveItem}
+                />
+                {size !== 'small' && (
+                  <Stack anchor="bottom-right">
+                    <Avatar background="blue!">DS</Avatar>
+                    <Box
+                      background="status-ok"
+                      pad={{ horizontal: 'xsmall' }}
+                      height="12px"
+                      width="12px"
+                      round
+                    />
+                  </Stack>
+                )}
+              </Box>
+            </Header>
+            <PageContent
+              gridArea="pageContent"
+              activeItem={activeItem}
+              setActiveItem={setActiveItem}
+            />
+          </AppContainer>
+        )}
+      </ResponsiveContext.Consumer>
+    </ResponsiveContext.Provider>
+  );
+};
+
+HeaderNavExample.propTypes = {
+  mobile: PropTypes.bool,
+};
+
+const AppContainer = ({ ...rest }) => {
+  const size = React.useContext(ResponsiveContext);
+
+  return (
+    <Box
+      background="background-back"
+      height={size === 'small' ? { max: 'large' } : undefined}
+      width={size === 'small' ? 'medium' : '100%'}
+      overflow="auto"
+      {...rest}
+    />
+  );
+};
+
+const MainNavigation = ({ activeItem, setActiveItem }) => {
+  const size = React.useContext(ResponsiveContext);
+
+  return size !== 'small' ? (
+    <Nav direction="row" gap="xsmall">
+      <NavItems activeItem={activeItem} setActiveItem={setActiveItem} />
+    </Nav>
+  ) : (
+    <Menu
+      icon={<MenuIcon />}
+      dropAlign={{ top: 'bottom', right: 'right' }}
+      hoverIndicator
+      items={pages.map((item, index) => ({
+        label: item.name,
+        onClick: () => setActiveItem(index),
+      }))}
+    />
+  );
+};
+
+MainNavigation.propTypes = {
+  activeItem: PropTypes.number,
+  setActiveItem: PropTypes.func,
+};
+
+const NavItems = ({ activeItem, setActiveItem }) => {
+  return (
+    pages &&
+    pages.map((item, index) => (
+      <SimpleButton
+        key={item.name}
+        name={item.name}
+        active={index === activeItem}
+        onClick={() => setActiveItem(index)}
+        round="xsmall"
+      />
+    ))
+  );
+};
+
+const SimpleButton = ({ active, name, ...rest }) => {
+  const [hover, setHover] = React.useState();
+
+  return (
+    <Button
+      onMouseOver={() => setHover(true)}
+      onFocus={() => setHover(true)}
+      onMouseOut={() => setHover(false)}
+      onBlur={() => setHover(false)}
+      {...rest}
```

Comment:
Not correct placement of comment, but should tooltips be turned off on mobile? Or be positioned differently? This would be cutoff if it were on a mobile screen:
<img width="195" alt="Screen Shot 2020-05-07 at 5 04 12 PM" src="https://user-images.githubusercontent.com/12522275/81355989-ec22ae00-9084-11ea-8251-ecb0e0630b01.png">

Reply:
I moved alignment direction -- can't think of harm having it left on for mobile.